modules/browsemap/src/main/java/com/sygic/modules/browsemap/viewmodel/BrowseMapFragmentViewModel.kt

```diff
@@ -29,17 +30,17 @@ import com.sygic.ui.common.sdk.utils.requestLocationAccess
 @AutoFactory
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
 class BrowseMapFragmentViewModel internal constructor(
-    @Assisted attributesTypedArray: TypedArray?,
     app: Application,
+    @Assisted initComponent: MapFragmentInitComponent,
+    private val mapDataModel: ExtendedMapDataModel,
     private val poiDataManager: PoiDataManager,
-    private val extendedMapDataModel: ExtendedMapDataModel,
     private val mapInteractionManager: MapInteractionManager,
     private val locationManager: LocationManager,
     private val permissionsManager: PermissionsManager
 ) : AndroidViewModel(app), MapInteractionManager.Listener, DefaultLifecycleObserver {
 
     @MapSelectionMode
-    var mapSelectionMode: Int = MapSelectionMode.MARKERS_ONLY
+    var mapSelectionMode: Int = MAP_SELECTION_MODE_DEFAULT_VALUE
```

Comment:
tym ze to inicializujes aj v konstruktore tu nemusi byt ten `= MAP_SELECTION_MODE_DEFAULT_VALUE`